pulsar-io/canal/src/main/java/org/apache/pulsar/io/canal/CanalAbstractSource.java

```diff
@@ -131,10 +124,11 @@ protected void process() {
                 connector.subscribe();
                 while (running) {
                     Message message = connector.getWithoutAck(canalSourceConfig.getBatchSize());
-                    // delete the setRaw in new version of canal-client
                     message.setRaw(false);
-                    List<FlatMessage> flatMessages = FlatMessage.messageConverter(message);
-                    long batchId = message.getId();
+                    SimpleCanalConnector con = (SimpleCanalConnector) connector;
+                    con.isLazyParseEntry();
```

Comment:
I am not very familiar with Canal. does `isLazyParseEntry()` changing the state of the connector, so the `con` will parse entries lazily?

Reply:
This line of code is useless. It should be generated automatically when changing files. I didn't notice it. Now I have removed it. Please review the code again.